hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/token/impl/CryptoPreTransactionHandlerImplTest.java

```diff
@@ -478,6 +503,21 @@ private TransactionBody cryptoDeleteAllowanceTransaction(final AccountID id) {
                 .build();
     }
 
+    private TransactionBody cryptoTransferTransaction(final AccountID id) {
+        if (id.equals(payer)) {
```

Comment:
```suggestion
        if (Objects.equals(payer, id)) {
```

Reply:
What's wrong with using equals on the class itself? It is overridden in `AccountID`.